src/main/java/gregtech/loaders/recipe/GTRecipeLoaders.java

```diff
@@ -0,0 +1,391 @@
+package gregtech.loaders.recipe;
+
+import gregtech.api.GTValues;
+import gregtech.api.recipes.loader.IRecipeLoader;
+
+import javax.annotation.Nonnull;
+
+public enum GTRecipeLoaders implements IRecipeLoader {
```

Comment:
I feel this either needs to be grouped by virtual categories or in alphabetic order. As right now for someone who is it not using everyday it is hard to find right category in all of these.